### ISSUE TITLE
test filestorage fails in Octave on Windows

### ISSUE THREAD

**Issue by AvinoamK** [kyamagu/mexopencv#292]
Using last build, test filestorage fails in Octave 4.2-rc3, in 
```
 function test_struct_array
            S = dir(mexopencv.root())  % Nx1 struct array
            SS = struct('dummy',S)    % wrap inside a scalar struct
            str = cv.FileStorage('dummy.xml', SS)
            [S2,~] = cv.FileStorage(str)
            S2 = S2.dummy  % unwrap
            S2 = [S2{:}]   % cell array of structs to 1xN struct array
            assert(isequal(S(:), S2(:)));
        end
```
since octave has extra fields, which get NaN on Windows:
```
   S(1).statinfo.blksize = NaN
   S(1).statinfo.blocks = NaN
``` 
and so S2, `isequall(NaN, NaN) `fails.
changing this line to 
            `assert(isequaln(S(:), S2(:)));`
fixes the problem.

**Comment by amroamroamro** [comment]
Thanks for the report. I'll include a fix in the next update.

**Comment by amroamroamro** [comment]
Fixed in #295
